packages/client/components/MeetingCard.tsx

```diff
@@ -0,0 +1,141 @@
+import styled from '@emotion/styled'
+import * as Sentry from '@sentry/browser'
+import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
+import {createFragmentContainer} from 'react-relay'
+import useRouter from '../hooks/useRouter'
+import {PALETTE} from '../styles/paletteV3'
+import {Elevation} from '../styles/elevation'
+import {Card} from '../types/constEnums'
+import getMeetingPhase from '../utils/getMeetingPhase'
+import {phaseLabelLookup} from '../utils/meetings/lookups'
+import {MeetingCard_meeting} from '../__generated__/MeetingCard_meeting.graphql'
+import useBreakpoint from '../hooks/useBreakpoint'
+import {Breakpoint} from '../types/constEnums'
+// import {ICON_SIZE} from '../styles/typographyV2'
+// import Icon from './Icon'
+
+const CardWrapper = styled('div')<{maybeTabletPlus: boolean}>(({maybeTabletPlus}) => ({
+  background: Card.BACKGROUND_COLOR,
+  borderRadius: Card.BORDER_RADIUS,
+  boxShadow: Elevation.CARD_SHADOW,
+  cursor: 'pointer',
+  flexShrink: 0,
+  maxWidth: '100%',
+  margin: maybeTabletPlus ? '0 16px 16px 0' : '0 0 16px',
+  transition: `box-shadow 100ms ease-in`,
+  width: maybeTabletPlus ? 320 : '100%',
+  ':hover': {
+    boxShadow: Elevation.CARD_SHADOW_HOVER
+  }
+}))
+
+const MeetingInfo = styled('div')({
+  padding: '12px 16px'
+})
+
+const Name = styled('div')({
+  color: PALETTE.SLATE_700,
+  fontSize: 20,
+  lineHeight: '32px'
+})
+
+const Meta = styled('div')({
+  color: PALETTE.SLATE_600,
+  fontSize: 14,
+  lineHeight: '24px'
+})
+
+const MeetingImgWrapper = styled('div')({
+  position: 'relative'
+})
+
+const MeetingTypeLabel = styled('div')({
+  color: PALETTE.WHITE,
+  fontSize: 11,
+  fontWeight: 600,
+  left: 16,
+  lineHeight: '16px',
+  position: 'absolute',
+  textTransform: 'uppercase',
+  top: 12
+})
+
+const MeetingImg = styled('img')({
+  borderRadius: `${Card.BORDER_RADIUS}px ${Card.BORDER_RADIUS}px 0 0`,
+  display: 'block',
+  overflow: 'hidden',
+  width: '100%'
+})
+
+interface Props {
+  meeting: MeetingCard_meeting
+}
+
+const ILLUSTRATIONS = {
+  retrospective: `${__STATIC_IMAGES__}/illustrations/retro-mtg-color-bg.svg`,
```

Comment:
this reminds me we should move away from this pattern to support airgapped deployments.
fine for now! i'll backlog an issue